app/src/main/java/com/droidfoundry/droidmetronome/control/ClickListenerModel.java

```diff
@@ -29,7 +28,6 @@ public ClickListenerModel(TextView descricaoSeekbar, int min, int max, Context c
     }
 
 
-
     SeekBar.OnSeekBarChangeListener seekBarListener = new SeekBar.OnSeekBarChangeListener() {
```

Comment:
Reduce this anonymous class number of lines from 31 to at most 20, or make it a named class (issue #170).